very_good_core/hooks/test/pre_gen_test.dart

```diff
@@ -40,6 +41,7 @@ void main() {
             'ios_application_id': 'app.id',
             'macos_application_id': 'app.id',
             'windows_application_id': 'app.id',
+            'current_year': 2024,
```

Comment:
Wondering if we can use the [clock](https://pub.dev/packages/clock) package as a dev dependency to fake the current time and avoid having the test fail in 2025.

```suggestion
            'current_year': 2024,
```

Reply:
Thank you for the suggestion! I've already added the clock package as a dev dependency to fake the current time for our tests. I hope this approach works well to prevent any issues in 2025. Let me know if there's anything else we should consider!